R/run_pacta.R

```diff
@@ -0,0 +1,28 @@
+#' Run PACTA for all portfolios matching a pattern in an input directory
+#'
+#' @param wd String. Working directory pointing to pacta's source code
+#' @param regext String. Regular expression that defines a valid portfolio name.
+#'
+#' @examples
+#' run_pacta()
+#' @noRd
+run_pacta <- function(wd = here::here()) {
+  withr::local_dir(wd)
+
+  portfolios <- portfolios()
+  command <- glue::glue("Rscript --vanilla web_tool_script_1.R {portfolios};\
+                          Rscript --vanilla web_tool_script_2.R {portfolios}")
+
+  for (i in seq_along(command)) {
+    message("Start portfolio: ", portfolios[[i]])
+    system(command[[i]])
```

Comment:
generally prefer `system2` over `system`, but only if that's actually the way to go.  https://ropensci.org/blog/2021/09/13/system-calls-r-package/